apps/api-gateway/src/verification/dto/request-proof.dto.ts

```diff
@@ -222,12 +218,13 @@ export class RequestProofDto extends ProofPayload {
   @IsNotEmpty({ message: 'ProofFormatDto must not be empty' })
   @Type(() => IndyDto)
   proofFormats?: IndyDto;    
```

Comment:
In next iteration maybe we can only accept `proofFormats?: IndyDto;` or `presentationDefinition?:ProofRequestPresentationDefinition;`